src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
```

Comment:
This class name does not match BEM schema (BlockName__ElementName). Use either `navigation__item` or `navigation__items-list-item`. Selectors in CSS files should be changed accordingly. You can reed more about BEM notation here: https://mate-academy.github.io/style-guides/htmlcss.html#oocss-and-bem